doc/programs.md

```diff
@@ -73,9 +73,9 @@
 - FILE, Chaosnet file server.
 - FIND, search for files.
 - FRETTY, display list of free TTYs.
-- FTPS, FTP Server.
-- FTPU, FTP Client.
-- GCMAIL, delete old files from .MAIL.
+- FTPS, FTP server.
+- FTPU, FTP client.
+- GCMAIL, delete old files from .MAIL.;.
```

Comment:
I don't think `;` is needed unless it's a full file name with FN1 etc.  But I haven't checked all files to see what the most common usage is.

Reply:
I was going for consistency with TMPKIL ("clean out old files in .TEMP.;."), but equally we could remove the semicolon from the latter...